src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,88 @@
+package homework_2.random_chars_table;
+
+import homework_2.random_chars_table.exception.RandomCharsTableException;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static homework_2.random_chars_table.utils.Constants.ERROR_MESSAGE;
+import static homework_2.random_chars_table.utils.Constants.MAX_CHAR;
+import static homework_2.random_chars_table.utils.Constants.MIN_CHAR;
+import static homework_2.random_chars_table.utils.Utils.getData;
+import static homework_2.random_chars_table.utils.Utils.printMessage;
+import static java.lang.System.lineSeparator;
+
+public class RandomCharsTable {
```

Comment:
![image](https://user-images.githubusercontent.com/58872509/126909666-c064f675-7ae0-4262-a443-8213f45d74d8.png)
First number is width, then length. There's an example in Homework

Reply:
Sorry! I've just fixed it!